adhocteam/Head-Start-TTADP
# [TTAHUB-278] grantee search page

## Description of change
Add a new page (feature flagged for admins only right now) for grantee records. Right now, the page only has a heading, a search box, and a region selector.

## How to test
When you are logged in as an admin, the site should have a "Grantees" link in the left side nav. The link should take you to /grantee-search. On that page, there should be a heading, a region selector, and a search form. The should appear functional but don't, as of yet, have any impact on the page itself.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-278


## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
